samples/highcharts/chartchooser/categorical-relationship-map-bubble/demo.html

```diff
@@ -0,0 +1,8 @@
+<script src="https://cdnjs.cloudflare.com/ajax/libs/proj4js/2.4.4/proj4.js"></script>
+<script src="https://code.highcharts.com/maps/highmaps.js"></script>
+<script src="https://code.highcharts.com/highcharts-more.js"></script>
+<script src="https://code.highcharts.com/maps/modules/exporting.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+
+<script src="https://code.highcharts.com/mapdata/countries/kr/kr-all.js"></script>
```

Comment:
Use [TopoJSON map instead](https://www.highcharts.com/docs/maps/map-collection#using-the-map-collection) (and remove the proj4.js file). See use cases in the map demos, especially the updated flight-routes demo.

Reply:
Done 👍